packages/contracts/test/plugin/plugin-setup-processor.ts

```diff
@@ -720,186 +882,164 @@ describe('Plugin Setup Processor', function () {
         ).to.be.revertedWith(customError('IPluginNotSupported', plugin));
       });
 
-      it('reverts if plugin supports IPlugin, but is non upgradable', async () => {
+      it('reverts if plugin supports the `IPlugin` interface, but is non-upgradable', async () => {
+        await targetDao2.grant(
```

Comment:
is the following 3 awaits needed ? seems like it only checks to revert if it's non upgradeable and this 3 await don't have any meaning and we can directly pass `targetDao` instead of `targetDao2`.

Reply:
No, good catch. We don't need them. This was refactored but now it is not needed anymore.